pkgs/by-name/to/tocpdf/package.nix

```
@@ -0,0 +1,40 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "tocpdf";
+  version = "0.3.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    pname = "tocPDF";
+    inherit version;
+    hash = "sha256-B+UcvyjWceVErf1uDyGGTGwKBCGHmSOF19Vbk15cPp8=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+    wheel
+  ];
+
+  dependencies = with python3Packages; [
+    click
+    pdfplumber
+    pypdf
+    tika
+    tqdm
+  ];
+
```

```suggestion

  # no test
  doCheck = false;

```
It's better to add a comment to mark that no test in this package.